### PR TITLE
Add Databricks autogen module

### LINKED ISSUE #10879: Initial support for DataBricks service

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Description

Fix #10879

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Databricks/Az.Databricks.csproj [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+-->
+    <!-- Some methods are marked async and don't have an await in them -->
+    <!--
+    <NoWarn>1998</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="PowerShellStandard.Library" Version="5.1.0" />
```
Comment (dolauli) [inline src/Databricks/Az.Databricks.csproj:24]: Please clean the comments.

File: src/Databricks/Databricks.sln [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29709.97
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Accounts", "..\Accounts\Accounts\Accounts.csproj", "{142D7B0B-388A-4CEB-A228-7F6D423C5C2E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Az.Databricks", "Az.Databricks.csproj", "{B6E9156B-F29F-41BE-A5E4-533A873C99B8}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Authentication", "..\Accounts\Authentication\Authentication.csproj", "{62843FE6-7575-4D88-B989-7DF7EEC0BC01}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Authentication.ResourceManager", "..\Accounts\Authentication.ResourceManager\Authentication.ResourceManager.csproj", "{442C609B-A431-4A71-B289-08F0B63C83E5}"
```
Comment (dolauli) [inline src/Databricks/Databricks.sln:11]: Why Accounts, Authentication, Authenticators, Authentication.ResourceManager are required?

File: src/Databricks/Az.Databricks.psd1 [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+# Generated on: 3/24/2020
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = './Az.Databricks.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
```
Comment (isra-fel) [inline src/Databricks/Az.Databricks.psd1:15]: Version usually starts from `0.1.0`
See https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

File: src/Databricks/ChangeLog.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Upcoming Release
+
+## Version 0.0.1
```
Comment (isra-fel) [inline src/Databricks/ChangeLog.md:22]: And here too :)

File: src/Databricks/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+[assembly: AssemblyTitle("Microsoft Azure Powershell - DataBricks")]
+[assembly: AssemblyCompany(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCompany)]
+[assembly: AssemblyProduct(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyProduct)]
+[assembly: AssemblyCopyright(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCopyright)]
+
+[assembly: ComVisible(false)]
+[assembly: CLSCompliant(false)]
+[assembly: Guid("4D3E68BD-35BC-4C97-8F40-A832CBA17AF7")]
+[assembly: AssemblyVersion("0.0.1")]
```
Comment (isra-fel) [inline src/Databricks/Properties/AssemblyInfo.cs:27]: Here too

File: src/Databricks/Databricks.sln (hunk 2 above)
Comment (dingmeng-xue) [reply src/Databricks/Databricks.sln:11]: Keep it there following convention.

**Comment by dingmeng-xue** [comment]
> `src/Databricks/MSSharedLibKey.snk` was this file chewcked in by accident?

Keep it there following current Autorest structure.

**Pull request merged**